cinema/serializers.py

```diff
@@ -45,10 +54,29 @@ class Meta:
         fields = ("id", "title", "description", "duration", "genres", "actors")
 
 
+class TicketSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Ticket
+        fields = ("id", "row", "seat", "movie_session")
+        validators = [
+            UniqueTogetherValidator(
+                queryset=Ticket.objects.all(),
+                fields=["seat", "row"]
+            )
+        ]
+
```

Comment:
There should be implemented `def validate` function in the serializer. UniqueTogether you can add to the model's Meta.